src/deluge/util/functions.cpp

```diff
@@ -1301,9 +1301,11 @@ void getThingFilename(char const* thingName, int16_t currentSlot, int8_t current
 }
 
 bool isAudioFilename(char const* filename) {
+	if (*filename == '.') {
```

Comment:
style nit: filename[0] is clearer 
```suggestion
	if (filename[0] == '.') {
```

Reply:
Fully agreed!